From f1f2d5bbd727cecec9dacf82eca4f4708662508a Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Mon, 18 Jul 2016 19:21:55 +0300
Subject: [PATCH 0604/2241] net: mvpp2x: set dev->tx_queue_length according to
 port tx queue length

- set dev->tx_queue_length according to port tx queue length
  instead of hardcoded to MVPP2_MAX_TXD

Change-Id: I7341bfbb8237e258e4a265d16ba48aaa901d258b
Signed-off-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/31225
Reviewed-by: Yuval Caduri <cyuval@marvell.com>
Verified-Armada8K: Yuval Caduri <cyuval@marvell.com>
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
index 02bf93b..8d29547 100644
--- a/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
+++ b/drivers/net/ethernet/marvell/mvpp2x/mv_pp2x_main.c
@@ -3828,7 +3828,7 @@ static int mv_pp2x_port_probe(struct platform_device *pdev,
 
 	port->num_tx_queues = mv_pp2x_txq_number;
 	port->num_rx_queues = mv_pp2x_rxq_number;
-	dev->tx_queue_len = MVPP2_MAX_TXD;
+	dev->tx_queue_len = tx_queue_size;
 	dev->watchdog_timeo = 5 * HZ;
 	dev->netdev_ops = &mv_pp2x_netdev_ops;
 	mv_pp2x_set_ethtool_ops(dev);
-- 
2.7.4

